juniper_official/Linecard/aft-server-entry-stats.rule

```diff
@@ -0,0 +1,41 @@
+healthbot {
+    topic external {
+        rule aft_server_entry_stats {
```

Comment:
Not that all name "topic", "rule", "field-names", "trigger/term" names everything should use "-"